conf/config.yaml

```diff
@@ -65,6 +65,11 @@ nginx_config:                     # config for render the template to genarate n
     client_header_timeout: 60s     # timeout for reading client request header, then 408 (Request Time-out) error is returned to the client
     client_body_timeout: 60s       # timeout for reading client request body, then 408 (Request Time-out) error is returned to the client
     send_timeout: 10s              # timeout for transmitting a response to the client.then the connection is closed
+  server:                          # set custusmized directives in nginx.conf server field  
+    server_tokens: on              # Enables emitting nginx version on error pages and in the "Server"
+    add_header: "API-Gateway-ID: '001'" # add custusmized header
+    underscores_in_headers: off    # Disable the use of underscores in client request header fields
+    client_body_buffer_size: 64k  # Sets buffer size for reading client request body
```

Comment:
the default value is `16k`, why change it?

Reply:
I think you can leave `server` section empty, and add comment configure, otherwise, you will effect other users.

Reply:
I will  delete it